app/templates/main/browse-series.html

```diff
@@ -58,286 +58,240 @@
+                                    <input class="govuk-input govuk-date-input__input govuk-input--width-2"
+                                           id="date_to_month"
+                                           name="date_to_month"
+                                           type="text"
+                                           value="{% if user_filters['date_range'] %}{% if user_filters['date_range']['date_to'] %}{{ user_filters['date_range']['date_to'][3:5] }}{% endif %}{% endif %}"
+                                           inputmode="numeric">
+                                </div>
                             </div>
-                        </div>
-                        <div class="govuk-date-input__item">
-                            <div class="govuk-form-group">
-                                <input class="govuk-input govuk-date-input__input govuk-input--width-4"
-                                       id="date-to-year"
-                                       name="date-to-year"
-                                       type="text"
-                                       inputmode="numeric">
-                                <label class="govuk-label govuk-date-input__label filters-form__date__label"
-                                       for="date-to-year">(YYYY)</label>
+                            <div class="govuk-date-input__item govuk-date-input__item--browse-all">
+                                <div class="govuk-form-group">
+                                    <label class="govuk-label govuk-date-input__label filters-form__date__label"
+                                           for="date_to_year">Year</label>
+                                    <input class="govuk-input govuk-date-input__input govuk-input--width-4"
+                                           id="date_to_year"
+                                           name="date_to_year"
+                                           type="text"
+                                           value="{% if user_filters['date_range'] %}{% if user_filters['date_range']['date_to'] %}{{ user_filters['date_range']['date_to'][6:10] }}{% endif %}{% endif %}"
+                                           inputmode="numeric">
+                                </div>
                             </div>
                         </div>
-                    </div>
-                </fieldset>
+                    </fieldset>
+                </div>
             </div>
-            <div class="filters-form__buttons">
-                <button type="button"
-                        class="govuk-button govuk-button__filters-form-apply-button"
-                        data-module="govuk-button">Apply filters</button>
-                <button type="button"
-                        class="govuk-button__filters-form-clear-button"
-                        data-module="govuk-button">Clear all filters</button>
+            <div class="browse-all-filter-container">
```

Comment:
can we rename trhe browse-all class since it is not unique to the browse all view